orttraining/orttraining/python/ort_trainer.py

```diff
@@ -681,6 +628,9 @@ def _init_session(self):
         if self.onnx_model_ is None:
             return
 
+        if self._enable_internal_postprocess:
+            self._onnx_model_ = postprocess.run_postprocess(self.onnx_model_)
+
```

Comment:
The call on lines 631-632 covers the scenario where user provides `self.onnx_model_` at `ORTTrainer` constructor. Note it only does internal post processing, but not extra post processing. This seems to be a pre-exsiting bug
Suggest adding the following on line 633
```python
if self.post_process_model_fn_:
                self.post_process_model_fn_(self.onnx_model_)
```